tests/app/dao/test_unsubscribe_request_dao.py

```diff
@@ -293,6 +295,50 @@ def test_assign_unbatched_unsubscribe_requests_to_report_dao(sample_service):
     assert len(unsubscribe_requests) == 2
 
 
+def test_unsubscribe_reports_only_includes_those_with_requests(sample_service):
+    unsubscribe_request_report_1 = UnsubscribeRequestReport(
+        id=uuid4(),
+        count=123,
+        earliest_timestamp=midnight_n_days_ago(4),
+        latest_timestamp=midnight_n_days_ago(0),
+        service_id=sample_service.id,
+    )
+    create_unsubscribe_request_reports_dao(unsubscribe_request_report_1)
```

Comment:
creating of an unsub report is quite lengthy, it would be good to make it into a function in `https://github.com/alphagov/notifications-api/blob/1f8e5b1bc2c966352ef7f13a082ffa5f3e0e8465/tests/app/db.py` to keep our tests easier to take in at a glance.